Public/Src/Tools/Execution.Analyzer/Args.cs

```diff
@@ -143,7 +143,7 @@ public Args(string[] args)
 
             // The fingerprint store based cache miss analyzer only uses graph information from the newer build,
             // so skip loading the graph for the earlier build
-            if (m_mode.Value != AnalysisMode.CacheMiss)
+            if (m_mode.Value != AnalysisMode.CacheMiss || m_mode.Value != AnalysisMode.BXLInvocationXLG)

```

Comment:
>m_mode.Value != AnalysisMode.BXLInvocationXLG) [](start = 58, length = 46)

we might want to think about a better way of doing this check. If you are going to add more analyzers that do not load the graph, listing all of them here might look a little bit ugly.

Reply:
Maybe an enum or list somewhere where we can add the "exception" analyzers

---
In reply to: [306044093](https://github.com/microsoft/BuildXL/pull/636#discussion_r306044093) [](ancestors = 306044093)